roles/ovirt-collect-logs/tasks/engine.yml

```
@@ -53,6 +53,11 @@
       src: "/var/log/openvswitch/ovsdb-server-nb.log"
       dest: "ovsdb-server-nb.log"
 
+    - src: "/var/log/ovn"
+      dest: "ovn-logs"
+        - src: "/var/log/openvswitch"
+          dest: "ovs-logs"
```

please fix the indentation
```suggestion
    - src: "/var/log/openvswitch"
      dest: "ovs-logs"
```